review-notifications/src/background.js

```diff
@@ -139,7 +144,14 @@ async function findMentioned(commentsUrl) {
 async function extractDataFromPR(prObject) {
   try {
     const mentionedUsers = await findMentioned(prObject.comments_url);
-    const comments = await extractComments(prObject);
+    const reviewComments = await extractComments(prObject.review_comments_url);
+    const prComments = await extractComments(prObject.comments_url);
+    const comments1 = [].concat.apply([], reviewComments.data);
```

Comment:
There is a method for flattening arrays, not sure if this will work but if it does it is much more readable than `concat`.
Try `reviewComments.data.flat()`